src/Components/EditTodo.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect } from "react";
+
+function EditTodo({ todo, saveEditedTodo, cancelEdit }) {
```

Comment:
editing should be handled in same add component. 
AddEditTodo.jsx should be able to handle both adding and editing todo @kaushaki-kumari